cimi-resources/src/com/sixsq/slipstream/ssclj/resources/service_attribute_namespace.clj

```diff
@@ -89,7 +89,7 @@
 (defmethod crud/add resource-name
   [{{:keys [prefix uri]} :body :as request}]
   (if-let [id (colliding-id prefix uri)]
-    (response/response-conflict id)
+    (response/ex-conflict id)
```

Comment:
What's the reason for this change?  For me, this causes the lifecycle tests for the Service Attribute Namespace to fail.

Reply:
It was failing for me with `response/response-conflict`, hence the change.  I might need to recompile the `response` dependency and retry.  I'll update you shortly.

Reply:
<img width="1109" alt="screen shot 2018-02-20 at 08 22 44" src="https://user-images.githubusercontent.com/353767/36411730-50935992-1617-11e8-8af9-b09020217945.png">

Reply:
Changed back to `response/response-conflict`.  All compiles and tests pass on my side.